ftp-source/src/main/java/org/springframework/cloud/stream/module/ftp/FtpSourceConfigurationProperties.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2015 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.ftp;
+
+import java.io.File;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author David Turanski
+ */
+@ConfigurationProperties
+public class FtpSourceConfigurationProperties {
+
+
+
+	private String remoteDir = "/";
```

Comment:
Add javadoc to all those fields, as this is what will end up in the `.json` metadata file.
Ditto for all other `*Properties` classes